### PR TITLE
[WINMINE] Rename WineMine to Minesweeper

### PR DESCRIPTION
**Pull request by Artem13327**
I also made a translation somewhere right away, as I was able to take it from the shortcuts.inf file and check it for correctness. I will also have to ask for help with line 327 in the shortcuts.inf file from those who know Czech, and with line 844 from those who know Indonesian

### CODE REVIEW

**Comment by ThFabba** [comment]
Does this fix something? If not, I'd rather leave it as WineMine, because that's what it is...

**Comment by Matriks404** [comment]
I agree with @Artem13327 name change, since it makes it more clear what game is about, although it definitely doesn't make that much of difference.

**Review by GeoB99 — CHANGES_REQUESTED** [review]
I'm definitely against this change because WineMine isn't ours (it is literally in the name) and I don't think the license explicitly allows it. Well, even though if it may allow it I don't think we're giving good impressions to Wine with this move.

File: boot/bootdata/CMakeLists.txt [hunk 1]
```diff
@@ -103,7 +103,7 @@ add_livecd_shortcut("Notepad" notepad.exe "Profiles/All Users/Start Menu/Program
 add_livecd_shortcut("WordPad" wordpad.exe "Profiles/All Users/Start Menu/Programs/Accessories")
 add_livecd_shortcut("Solitaire" sol.exe "Profiles/All Users/Start Menu/Programs/Games")
 add_livecd_shortcut("Spider Solitaire" spider.exe "Profiles/All Users/Start Menu/Programs/Games")
-add_livecd_shortcut("WineMine" winmine.exe "Profiles/All Users/Start Menu/Programs/Games")
+add_livecd_shortcut("Minesweeper" winmine.exe "Profiles/All Users/Start Menu/Programs/Games")
```
Comment (binarymaster) [inline boot/bootdata/CMakeLists.txt:106]: If you limit the changes to only renaming the shortcuts, I think this could be acceptable.

**Comment by MesaBlack** [comment]
https://www.youtube.com/watch?v=NIgfiSzCy1o

**Pull request closed without merging**